pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/logging/LogUtil.java

```diff
@@ -1,5 +0,0 @@
-package org.broadinstitute.ddp.logging;
-
-public class LogUtil {
```

Comment:
Moved to [ddp-common](https://github.com/broadinstitute/ddp-study-server/pull/2854/files#r1570800530) so both DSM and DSS can use the same class.